test/integration.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2021 Benjamin Davidson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Benjamin Davidson
+-/
+
+import analysis.special_functions.integrals
+open interval_integral real
+open_locale real
+
+/-- constants -/
+example : ∫ x:ℝ in 8..11, (1:ℝ) = 3 := by norm_num
+example : ∫ x:ℝ in 5..19, (12:ℝ) = 168 := by norm_num
```

Comment:
Usually we put spaces around colons in binders and casts:

```suggestion
example : ∫ x : ℝ in 8..11, (1 : ℝ) = 3 := by norm_num
example : ∫ x : ℝ in 5..19, (12 : ℝ) = 168 := by norm_num
```

same throughout the file

Reply:
Ah yes of course. Thanks!